Makefile

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+include makefiles/dependency.mk
```

Comment:
is there any reason we need this? Does the current make file not support golint-ci and staticcheck?